src/scripts/browserdeviceprofile.js

```diff
@@ -592,6 +593,15 @@ define(['browser'], function (browser) {
             });
         }
 
+        if (canPlayAv1) {
+            profile.DirectPlayProfiles.push({
+                Container: 'webm',
+                Type: 'Video',
+                AudioCodec: webmAudioCodecs.join(','),
+                VideoCodec: 'AV1'
+            });
+        }
```

Comment:
I think we could merge these blocks by collecting supported video codecs in some array (`webmVideoCodecs`).